docs/examples/contrib/piccolo/piccolo_conf.py

```diff
@@ -1,3 +1,3 @@
 from piccolo.engine.sqlite import SQLiteEngine
 
-DB = SQLiteEngine()
+DB = SQLiteEngine(path=":memory:?cache=shared")
```

Comment:
When I run this locally, it creates a file `:memory:?cache=shared` instead of using an in-memory DB. 

Maybe we should just revert this..

Reply:
This shouldn't happen (at least in my case) because we remove the file using `Path(DB.path).unlink()` after the tests is completed. I agree with you that we should use `DB = SQLiteEngine()` and as I wrote in the previous comment this will set the default `SQliteEngine` and then delete the created `piccolo.sqlite`, after completing the tests, with `Path(DB.path).unlink()`